README.md

```diff
@@ -1,9 +1,11 @@
 # posh-git
 
 [![master build status](https://ci.appveyor.com/api/projects/status/eb8erd5afaa01w80/branch/master?svg=true&pendingText=master%20%E2%80%A3%20pending&failingText=master%20%E2%80%A3%20failing&passingText=master%20%E2%80%A3%20passing)](https://ci.appveyor.com/project/dahlbyk/posh-git/branch/master)
+[![develop pscore build status](https://travis-ci.org/dahlbyk/posh-git.svg?branch=master)](https://travis-ci.org/dahlbyk/posh-git)
```

Comment:
This is going to conflict with the PR I've got that is a rewrite for the README for the 1.0.0 release.  AFAIK - master which is currently 0.7.2 doesn't need a Travis build since that version will only ever run on Windows.  Eventually 0.7.x will get migrated to a maintenance branch and develop (1.x) will move to master.  At that point, we will want the Travis builds on master.

Reply:
Oh yeah,, I've got the develop badge link integrated into the PR - see https://github.com/dahlbyk/posh-git/pull/529